l10n-pocket/en/brands.ftl

```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### Please read the trademarks localization guide
+### https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#brands-copyright-and-trademark
+
+## Company names
+
+-brand-name-mozilla = Mozilla
+-brand-name-pocket = Pocket
+-brand-name-read-it-later-inc = Read It Later, Inc.
+
+## Pocket product
+
+-brand-name-pocket-premium = Pocket Premium
+-brand-name-pocket-for-firefox = Pocket for Firefox
```

Comment:
Is this combination definitely a brand name in its own right? I can see locales wanting to translate `for` here more often than not.

Reply:
I wondered about this and thought that putting it all in one string gives the translators more flexibility for word order than stipulating that a translation of "for" has to go in the middle.

Can revert, though!

Reply:
Brand name variables are typically thing that shouldn't be translated (but we keep them in FTL land to still give the option vs keep them as external variables). 

My gut says this should probably be a regular FTL string e.g. `{ -brand-name-pocket } for `{ -brand-name-firefox }`. Translators still have the same flexibility.